test/framed/std/io_test.clj

```diff
@@ -12,14 +12,25 @@
     (is (= expected (std.io/path-join "foo/" "/bar" "quux.txt")))))
 
 (deftest test-stream-copy
-  (let [contents "hello"
-        f1 (std.io/spit (std.io/tempfile) contents)
-        f2 (std.io/tempfile)]
-    (is (empty? (slurp f2)))
-    (with-open [istream (io/input-stream f1)
-                ostream (io/output-stream f2)]
-      (std.io/stream-copy istream ostream))
-    (is (= contents (slurp f2)))))
+  (let [contents "hello"]
+    (testing "with explicit bufsize"
+      (let [
+            f1 (std.io/spit (std.io/tempfile) contents)
+            f2 (std.io/tempfile)
+            bufsize 2]
+        (is (empty? (slurp f2)))
+        (with-open [istream (io/input-stream f1)
+                    ostream (io/output-stream f2)]
+          (std.io/stream-copy bufsize istream ostream))
+        (is (= contents (slurp f2)))))
+    (testing "with default bufsize"
+      (let [f1 (std.io/spit (std.io/tempfile) contents)
+            f2 (std.io/tempfile)]
+        (is (empty? (slurp f2)))
+        (with-open [istream (io/input-stream f1)
+                    ostream (io/output-stream f2)]
+          (std.io/stream-copy istream ostream))
+        (is (= contents (slurp f2)))))))
```

Comment:
This seems like it's literally duplicated and could be DRY'd?

Reply:
I didn't come up with a DRY solution I liked. A helper function that accepts a `do-copy` function ..? Tricky when the arities differ like this